### PR TITLE
fix: add Exoutia

### LINKED ISSUE #1247: Exoutia
## Profile Screenshot

image

### PR DESCRIPTION
**Pull request by Exoutia**
resolves #1247 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]
It seems like there is a prettier error. Please follow 2 space indentation rather than 4

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by Exoutia** [comment]
Ok

**Review by schmelto — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by kaiwalyakoparkar — APPROVED** [review]
lgtm 🚀 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by Exoutia** [comment]
@kaiwalyakoparkar but it's still failing prettier test 🥲🥲

**Comment by kaiwalyakoparkar** [comment]
> @kaiwalyakoparkar but it's still failing prettier test 🥲🥲

I re-run the action but it still says failed. This might be an action issue. Don't worry once confirmed your pr will be merged :)

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

File: profiles/Exoutia.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/94464106/180158833-d731794f-9099-4be7-a118-044c90bb1317.png",
```
Comment (schmelto) [inline profiles/Exoutia.json:2]:
```suggestion
  "image": "https://user-images.githubusercontent.com/94464106/180158833-d731794f-9099-4be7-a118-044c90bb1317.png",
```

File: profiles/Exoutia.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/94464106/180158833-d731794f-9099-4be7-a118-044c90bb1317.png",
+  "issueId": 1247,
+  "name": "Bibek Jha",
+  "username": "Exoutia"
```
Comment (schmelto) [inline profiles/Exoutia.json:5]:
```suggestion
  "issueId": 1247,
  "name": "Bibek Jha",
  "username": "Exoutia"
```

File: profiles/Exoutia.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/94464106/180158833-d731794f-9099-4be7-a118-044c90bb1317.png",
+  "issueId": 1247,
+  "name": "Bibek Jha",
+  "username": "Exoutia"
+}
```
Comment (schmelto) [inline profiles/Exoutia.json:6]:
```suggestion
{
  "image": "https://user-images.githubusercontent.com/94464106/180158833-d731794f-9099-4be7-a118-044c90bb1317.png",
  "issueId": 1247,
  "name": "Bibek Jha",
  "username": "Exoutia"
}

```

**Pull request merged**